src/modules/homepage.js

```diff
@@ -0,0 +1,54 @@
+const getData = async () => {
+  const request = await fetch('https://api.tvmaze.com/shows');
+  const response = await request.json();
+  return response;
+};
```

Comment:
[OPTIONAL] The `getData` function does not have any error handling ❌ , which means if there is an issue with the API call, the function will fail silently 🤫 . You can add a try-catch block to handle errors and provide useful feedback to the user. 🤓 it could go like below:

```
const getData = async () => {
  try {
    const request = await fetch('https://api.tvmaze.com/shows');
    if (!request.ok) {
      throw new Error(`Request failed with status ${request.status}`);
    }
    const response = await request.json();
    return response;
  } catch (error) {
    console.error('Error fetching data:', error);
    return [];
  }
};
```
Keep in mind that this is just an example, and it might need some adaptation 😉